.github/workflows/branch-updates.yml

```
@@ -22,76 +21,33 @@ jobs:
+        uses: astral-sh/setup-uv@v7
       - name: Install the project
         run: uv sync --all-extras --dev
       - name: Lint and Format with ruff
         run: |
           uv run ruff check
-          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          uv run ruff format
+          uv run ruff format --check
 
 
 
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)